### PR TITLE
Update CKBox & Image toolbar guides

### PR DESCRIPTION
**Pull request by godai78**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Update CKBox & Image toolbar guides. Closes https://github.com/cksource/ckeditor5-commercial/issues/5632

---

### Additional information

_Waiting for [15209](https://github.com/ckeditor/ckeditor5/pull/15209) to merge first. Update from base branch afterwards._

### CODE REVIEW

File: packages/ckeditor5-ckbox/docs/features/ckbox.md [hunk 1]
```diff
@@ -30,12 +30,12 @@ As a full-fledged file manager, CKBox also replaces the basic CKEditor&nbsp;5 im
 * Enables drag & drop uploads of images and other files.
 * Transforms the _Image_ toolbar button, allowing the user to quickly upload and insert an image without opening the CKBox UI.
 * Adds a separate dedicated toolbar button to open the CKBox UI to manage and reuse uploaded files.
-* Provides basic editing capabilities, like crop, resize, rotate, and flip.
+* Provides {@link @ckbox features/images/editing basic editing capabilities}, like crop, resize, rotate, and flip right from the {@link features/images-overview#image-contextual-toolbar image contextual toolbar}.
```
Comment (gorzelinski) [inline packages/ckeditor5-ckbox/docs/features/ckbox.md:33]: It's a nitpick, but "provides" sounds a little redundant here to me.

File: packages/ckeditor5-ckbox/docs/features/ckbox.md [hunk 2]
```diff
@@ -83,18 +83,20 @@ These plugins are, by default, loaded with predefined builds, such as `ClassicEd
 
 Please also remember, that the CKBox plugin requires the following dependency plugins to work properly: `ArticlePluginSet`, `PictureEditing`, `ImageUpload`, and `CloudServices`. Except for `ImageUpload` which is, likewise, available in predefined builds, these need to be added manually.
 
-Finally, add {@link module:ckbox/ckbox~CKBox} to your plugin list and [configure](#configuration) the feature as needed. An example configuration may look like this:
+If you want to be able to use CKBox image editing capabilities from within CKEditor&nbsp;5, you need to also include the `CKBoxImageEdit` plugin.
```
Comment (gorzelinski) [inline packages/ckeditor5-ckbox/docs/features/ckbox.md:86]: Maybe I would reverse the order of this sentence to something like:
> You must include the `CKBoxImageEdit` plugin if you want to use CKBox image editing capabilities from within CKEditor 5.

File: packages/ckeditor5-ckbox/docs/features/ckbox.md [hunk 3]
```diff
@@ -106,6 +108,23 @@ ClassicEditor
 
 Further in the document, the dependency plugins will be omitted in code listings for clarity.
 
+### Adding the image editing button to the image toolbar
+
+To invoke the CKBox image editor straight from the image contextual toolbar (availabnle at right click on an image), it needs to be added to the toolbar configuration.
```
Comment (gorzelinski) [inline packages/ckeditor5-ckbox/docs/features/ckbox.md:113]: Typos: "available", "right-click".

File: packages/ckeditor5-ckbox/docs/features/ckbox.md [hunk 4]
```diff
@@ -106,6 +108,23 @@ ClassicEditor
...
+
+{@img assets/img/toolbar-items.png 402 An extended contextual toolbar.}
+
+The snippet below show an example image contextual toolbar configuration.
```
Comment (gorzelinski) [inline packages/ckeditor5-ckbox/docs/features/ckbox.md:117]: Typos: "shows".

File: packages/ckeditor5-ckbox/lang/contexts.json [hunk 5]
```diff
@@ -1,5 +1,6 @@
 {
   "Open file manager": "Toolbar button tooltip for opening a file browser that allows inserting an image or a file to the editor.",
   "Cannot determine a category for the uploaded file.": "A message is displayed when CKEditor 5 cannot associate an image with any of the categories defined in CKBox while uploading an asset.",
-  "Cannot access default workspace.": "A message is displayed when the user is not authorised to access CKBox workspace configured as default one."
```
Comment (gorzelinski) [inline packages/ckeditor5-ckbox/lang/contexts.json:4]: Inconsistencies: "authorized" vs. "authorised".
Missing articles: "the CKBox workspace", "the default one".

File: packages/ckeditor5-image/docs/features/images-installation.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ ClassicEditor
 
 Inline images can be inserted in the middle of a paragraph or a link just like regular text. Block images, on the other hand, can be inserted only between other blocks like paragraphs, tables, or media. Being larger and existing as standalone content, block images can also have individual captions. Other than that, both types of images can be resized, linked, etc.
 
-By default, the {@link module:image/image~Image} plugin available in all {@link installation/getting-started/predefined-builds ready-to-use editor builds} provides support for both inline and block images, working as a glue for {@link module:image/imageinline~ImageInline} and {@link module:image/imageblock~ImageBlock} plugins:
+By default, the {@link module:image/image~Image} plugin available in all {@link installation/getting-started/predefined-builds predefined editor builds} provides support for both inline and block images, working as a glue for {@link module:image/imageinline~ImageInline} and {@link module:image/imageblock~ImageBlock} plugins:
```
Comment (gorzelinski) [inline packages/ckeditor5-image/docs/features/images-installation.md:53]: A little wordy to me. Maybe:
> By default, the Image plugin available in all ready-to-use editor builds supports both inline and block images, working as a glue for ImageInline and ImageBlock plugins.

File: packages/ckeditor5-image/docs/features/images-overview.md [hunk 7]
```diff
@@ -37,7 +37,8 @@ The [`@ckeditor/ckeditor5-image`](https://www.npmjs.com/package/@ckeditor/ckedit
 * {@link features/images-resizing Image resizing} lets the user control the dimensions of images in the content.
 * {@link features/images-linking Linking images} makes it possible to use them as URL anchors.
 * A selection of {@link features/image-upload image upload methods} allows for the most convenient way of adding images. These include support for {@link features/images-inserting#inserting-images-via-pasting-a-url-into-the-editor inserting an image via a URL} and even {@link features/images-inserting#inserting-images-via-a-source-url via pasting a URL into the editor} along with custom integrations.
-* Support for {@link features/images-responsive responsive images} in CKEditor&nbsp;5 is brought by the {@link features/ckbox CKBox} management platform. Responsive images will display properly on any viewport, enhancing the accessibility, reach and user experience.
+* Support for {@link features/images-responsive responsive images} in CKEditor&nbsp;5 is brought by the {@link features/ckbox CKBox} management platform. Responsive images will display properly on any viewport, enhancing the accessibility, reach, and user experience.
```
Comment (gorzelinski) [inline packages/ckeditor5-image/docs/features/images-overview.md:40]: Maybe we could avoid the passive voice:
> The CKBox management platform supports responsive images in CKEditor 5. Responsive images will display correctly on any viewport, enhancing the accessibility, reach, and user experience.

**Review by gorzelinski — COMMENTED** [review]
I found some typos and added some suggestions. Nothing major.

File: packages/ckeditor5-ckbox/lang/contexts.json (hunk 5 above)
Comment (godai78) [reply packages/ckeditor5-ckbox/lang/contexts.json:4]: This is part of https://github.com/ckeditor/ckeditor5/pull/15209, won't be addressing it here.

@pszczesniak have a look.

File: packages/ckeditor5-ckbox/docs/features/ckbox.md (hunk 2 above)
Comment (godai78) [reply packages/ckeditor5-ckbox/docs/features/ckbox.md:86]: In commit.

File: packages/ckeditor5-ckbox/docs/features/ckbox.md (hunk 3 above)
Comment (godai78) [reply packages/ckeditor5-ckbox/docs/features/ckbox.md:113]: In commit.

File: packages/ckeditor5-image/docs/features/images-installation.md (hunk 6 above)
Comment (godai78) [reply packages/ckeditor5-image/docs/features/images-installation.md:53]: In commit.

File: packages/ckeditor5-image/docs/features/images-overview.md (hunk 7 above)
Comment (godai78) [reply packages/ckeditor5-image/docs/features/images-overview.md:40]: In commit.

File: packages/ckeditor5-ckbox/docs/features/ckbox.md (hunk 1 above)
Comment (godai78) [reply packages/ckeditor5-ckbox/docs/features/ckbox.md:33]: Let's keep it, makes perfect sense information-wise.

**Pull request merged**